sdks/uniswapx-sdk/src/trade/PriorityOrderTrade.ts

```diff
@@ -40,6 +50,11 @@ export class PriorityOrderTrade<
   public get inputAmount(): CurrencyAmount<TInput> {
     if (this._inputAmount) return this._inputAmount;
 
+    // If we have classic quote data use that, otherwise use the order input amount
+    if(this.classicAmounts?.classicAmountInGasAndPortionAdjusted) {
+      return this.getClassicAmountIn();
+    }
```

Comment:
this probably takes precedence over `if (this._inputAmount) return this._inputAmount;`?